web_filter_header_button/demo/ir_module_module_view.xml

```diff
@@ -14,6 +14,11 @@
                     name="context"
                 >{'shown_in_panel': {'icon': 'fa-toggle-off', 'hotkey': 'u'}}</attribute>
             </filter>
+            <filter name="state" position="attributes">
+                <attribute
+                    name="context"
+                >{'group_by':'state', 'shown_in_panel': {'icon': 'fa-th-list', 'hotkey': 's'}}</attribute>
```

Comment:
Why do you need to put `group_by` here? What happen if a field is not present as `<filter>` in the search bar? Isn't possible to have it on the header but not on the search? Maybe `invisible="1"`?

Reply:
What about this?

Reply:
Yes, it's something I'd liked to have, but I didn't find a very obvious way to do it at first glance. If you set the invisble flag, the filter goes away from the webclient memory and when a query for the list I don't have it to be shown...

In the other hand working with the context key, has these limitations in groups, as if you inherit from an existing group filter you'll have to overwrite the context...

And you can't add a new independent attribute, as the view validator will avoid it...

Reply:
OK, please explain all of this in ROADMAP.rst.